go/kbfs/kbfshash/hash.go

```diff
@@ -55,6 +55,9 @@ const (
 	SHA256Hash HashType = 1
 	// SHA256HashV2 is the type of a SHA256 hash over V2-encrypted data.
 	SHA256HashV2 HashType = 2
+
+	// MaxHashType is the numerically-largest hash type.
+	MaxHashType HashType = SHA256HashV2
```

Comment:
Is it the the largest as in all bytes in the hash are 0xff like `MaxDefaultHash`, or is it the largest supported hash version? The comment sounds like former, but making it same value as V2 feels like latter.

Reply:
Oh maybe it's the hash type of MaxDefaultHash?

Reply:
It's the maximum defined numeric value of `HashType`.  I tried to be clear about that in the comment -- how can I improve it?

`MaxDefaultHash` is the actual maximum hash value itself, excluding the type (which needs to be prepended).

Reply:
Oh I see; how about "highest supported hash version"?